### ISSUE TITLE
supplemental repos file fails on Windows when a repo exists in both the repos file and the supplemental repos file

### ISSUE THREAD

**Issue by mikaelarguedas** [ros2/ci#200]
* Linux [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_linux&build=4846)](http://ci.ros2.org/job/ci_linux/4846/)
* Linux-aarch64 [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_linux-aarch64&build=1699)](http://ci.ros2.org/job/ci_linux-aarch64/1699/)
* macOS [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_osx&build=4024)](http://ci.ros2.org/job/ci_osx/4024/)
* Windows [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_windows&build=4890)](http://ci.ros2.org/job/ci_windows/4890/)

**Comment by emersonknapp** [comment]
I am seeing this issue too when trying to use a supplemental repos instead of a modified ros2.repos

https://ci.ros2.org/job/ci_windows/9638/console

```
12:54:00 ==> vcs import "src" --force --retry 5 --input 01-pluginlib.repos
12:54:00 === src\ros/pluginlib (git) ===
12:54:00 Invocation of command 'import' on client 'git' failed: PermissionError: [WinError 5] Access is denied: 'src\\ros/pluginlib' (c:\python37\lib\site-packages\vcstool\clients\git.py:242)
12:54:00 <== 'vcs import "src" --force --retry 5 --input 01-pluginlib.repos' exited with return code '1'
```

Perhaps this is a `vcs` bug instead of a ci bug though?

**Comment by hidmic** [comment]
Already being tracked in upstream `vcs`, see https://github.com/dirk-thomas/vcstool/issues/130.

**Comment by dirk-thomas** [comment]
This will be resolved by dirk-thomas/vcstool#134.
